README.md

```diff
@@ -2,133 +2,54 @@
-| [IRC](https://github.com/sockethub/sockethub-platform-irc) | <kbd>![completed](http://sockethub.org/res/img/checkmark.png) send</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png) join</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png) leave</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png) observe</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png) update</kbd> |
-
-## Out of date
-
-Implemented in a previous major release of Sockethub, never ported to the current release and unmaintained.
+**For more information on Sockethub see [Sockethub README](packages/sockethub/README.md)**
 
-| Platform (@context) | Type (@type) |
-| ------------------- | ------------ |
-| [Email](https://github.com/sockethub/sockethub-platform-email) *(SMTP, IMAP)* *to port* | <kbd>![completed](http://sockethub.org/res/img/checkmark.png) send</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png) fetch (imap)</kbd> |
-| [Facebook](https://github.com/sockethub/sockethub-platform-facebook) *to port* | <kbd>![completed](http://sockethub.org/res/img/checkmark.png) post</kbd> <kbd>send</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png)fetch</kbd> <kbd>request-friend</kbd> <kbd>remove-friend</kbd> <kbd>make-friend</kbd> <kbd>like</kbd> |
-| [Twitter](https://github.com/sockethub/sockethub-platform-twitter) *to port* | <kbd>![completed](http://sockethub.org/res/img/checkmark.png) post</kbd> <kbd>follow</kbd> <kbd>unfollow</kbd> <kbd>send</kbd> <kbd>![completed](http://sockethub.org/res/img/checkmark.png) fetch</kbd> |
+## Platforms
 
-## Future consideration
+* [Feeds](packages/sockethub-platform-feeds) *(RSS, Atom)*
+* [IRC](packages/sockethub-platform-irc) 
+* [XMPP](packages/sockethub-platform-xmpp) 
 
-List of possible future platforms and their potentials verb types.
+## Setup
 
-| Platform (@context) | Type (@type) |
-| ------------------- | ------------ |
-| pump.io | <kbd>post</kbd> <kbd>follow</kbd> <kbd>unfollow</kbd> <kbd>send</kbd> <kbd>fetch</kbd> |
-| WhatsApp | <kbd>send</kbd> <kbd>request-friend</kbd> <kbd>remove-friend</kbd> <kbd>make-friend</kbd> |
-| WebFinger | <kbd>search</kbd> |
-| RDF | <kbd>search</kbd> <kbd>fetch</kbd> |
-| Bitcoin (e.g. sending signed transactions to a node) | *(to be evaluated)* <kbd>send</kbd> <kbd>receive</kbd> ... |
-| Ethereum (e.g. contract events) | *(to be evaluated)* |
-| WebRTC (signaling) | *(to be evaluated)* |
-| Signal | *(to be evaluated)* |
-| [MicroPub](https://indieweb.org/micropub) (IndieWeb, ...) | *(to be evaluated)* |
-| [OStatus](https://en.wikipedia.org/wiki/OStatus) (GNUSocial, Mastodon, ...) | *(to be evaluated)* |
+### Dependencies
 
-# Setup
+```$ yarn run install```
 
-`$ yarn install`
+### Tests
 
-# Running
+```$ yarn run test```
 
-For development purposes, with examples enabled, run:
+### Run
 
-`$ DEBUG=sockethub* yarn run dev`
+```$ yarn run start```
```

Comment:
Shouldn't this rather document how to run Sockethub from a normal npm-installed module's executable? Running directly from a source repo is something you'd usually do for development, but not production.